.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+## Contributing to Resource Hub
+
+### Reporting a bug:
+
+Check if the bug was already reported in the [Github Issues](https://github.com/resource-hub-dev/rhub-api/issues).
+
+If it is not there, feel free to open a [new issue](https://github.com/resource-hub-dev/rhub-api/issues/new/choose). Make sure to follow
+the template and include all relevant information.
+
+If you want to:
+* suggest some new feature/functionality to the team
+* discuss some new feature/functionality you want to implement before starting the actual development
+* ask any question related to features/functionalities
+
+then please follow the appropriate template while creating the issue, and provide as much information as possible.
+
+### Sending a pull request with a patch/feature/fix:
+
+To send a pull request:
+* Make sure to have opened a bug report / feature request / question before, so it can be related and validated
```

Comment:
Suggested change:
Make sure a bug report / feature request / question was previously opened so it can be related and validated.